skycoin/skywire
# Advertise public visors.

Did you run `make format && make check`?
yes
Fixes #574 

 Changes:	
-	Add ability to register as the public visor in service discovery and connect on startup to 5 public visors. 

How to test this PR:
To test this pull request you need to check out to branch `advertise_public_visors` three next repositories `skywire-service-proxy-discovery`, `skywire`, `skywire-services`. 

Then you need to run the test environment from `skywire-services` and check if visors have new stcp transports on their dedicated pages https://localhost:8000/#/nodes/list/1
